Is in master so can be deleted with next upgrade (1.1.7.3+).

Taken from https://github.com/xerial/snappy-java/commit/9df6ed7bbce88186c82f2e7cdcb7b974421b3340
_________________________________________________________________
--- a/build.sbt
+++ b/build.sbt
@@ -55,7 +55,7 @@
     </scm>
 }
 
-scalaVersion in ThisBuild := "2.12.4"
+scalaVersion in ThisBuild := "2.12.8"
 
 javacOptions in (Compile, compile) ++= Seq("-encoding", "UTF-8", "-Xlint:unchecked", "-Xlint:deprecation", "-source", "1.7", "-target", "1.7")
 
--- a/project/build.properties
+++ b/project/build.properties
@@ -1,2 +1,2 @@
-sbt.version=1.0.3
+sbt.version=1.2.8

--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -1,8 +1,8 @@
-addSbtPlugin("com.github.gseitz" % "sbt-release"  % "1.0.6")
-addSbtPlugin("org.xerial.sbt"    % "sbt-sonatype" % "2.0")
-addSbtPlugin("com.jsuereth"      % "sbt-pgp"      % "1.1.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release"  % "1.0.11")
+addSbtPlugin("org.xerial.sbt"    % "sbt-sonatype" % "2.3")
+addSbtPlugin("com.jsuereth"      % "sbt-pgp"      % "2.0.0-M2")
 addSbtPlugin("com.github.sbt"    % "sbt-findbugs" % "2.0.0")
-addSbtPlugin("com.github.sbt"    % "sbt-jacoco"   % "3.0.3")
-addSbtPlugin("com.typesafe.sbt"  % "sbt-osgi"     % "0.9.2")
-addSbtPlugin("io.get-coursier"   % "sbt-coursier" % "1.0.0-RC13")
-addSbtPlugin("com.geirsson"      % "sbt-scalafmt" % "1.3.0")
+addSbtPlugin("com.github.sbt"    % "sbt-jacoco"   % "3.1.0")
+addSbtPlugin("com.typesafe.sbt"  % "sbt-osgi"     % "0.9.4")
+addSbtPlugin("io.get-coursier"   % "sbt-coursier" % "1.1.0-M7")
+addSbtPlugin("com.geirsson"      % "sbt-scalafmt" % "1.5.1")
 
